payload/platformsh/.platform/services.yaml

```diff
@@ -1,5 +1,22 @@
 mysqldb:
-    type: 'mysql:10.1'
+    type: mysql:10.2
```

Comment:
We are staying with 10.1 atm because of issue with legacy, see also system requirements page on supported section.